examples/go-code-tutorial/.gitignore

```diff
@@ -0,0 +1,5 @@
+/bazel-bazel-gazelle
```

Comment:
this file is redundant with the root .gitignore, is there a reason for that?

Reply:
it is part of the tutorial.  I talk about setting up the ignore file

Reply:
recommend: add comment to the .gitignore file to make it clear why it exists. It's counter intuitive (per Alex's response) so pro-actively avoid a repeat question/confusion for future reviewers.

Reply:
Will do!

Reply:
Done